### PR TITLE
indexer ops: pkg version number and reset DB option

### PR DESCRIPTION
**Pull request by gegaowp**
## Description 

- use the sui workspace version number
- add reset DB option to indexer config
- minor refactoring

## Test Plan 

- CI
- local run indexer and verify
```
**2023-03-21T12:50:28.449881Z  INFO sui_indexer::utils: Resetting database ...
2023-03-21T12:50:28.449945Z  INFO sui_indexer::utils: Dropping all tables in the database
2023-03-21T12:50:28.490907Z  INFO sui_indexer::utils: Dropped all tables in the database
2023-03-21T12:50:28.526135Z  INFO sui_indexer::utils: Reset database complete.**
2023-03-21T12:50:28.527464Z  INFO sui_indexer::store::pg_indexer_store: Found 2 tables with partitions : [{"objects_history": 0, "owner_history": 0}]
2023-03-21T12:50:28.540199Z  INFO sui_json_rpc: acl=Const("*")
2023-03-21T12:50:28.541523Z  INFO sui_json_rpc: Compatibility method routing enabled.
2023-03-21T12:50:28.541875Z  INFO sui_json_rpc: Sui JSON-RPC server listening on 0.0.0.0:3030 
**2023-03-21T12:50:28.542032Z  INFO sui_indexer: Sui indexer of version "0.29.0" started...**
```

### CODE REVIEW

File: crates/sui-cluster-test/src/cluster.rs [hunk 1]
```diff
@@ -206,11 +206,10 @@ impl Cluster for LocalNewCluster {
                 rpc_server_url: indexer_address.as_ref().unwrap().ip().to_string(),
                 rpc_server_port: indexer_address.as_ref().unwrap().port(),
                 migrated_methods: IndexerConfig::all_migrated_methods(),
+                reset_db: true,
```
Comment (wlmyng) [inline crates/sui-cluster-test/src/cluster.rs:209]: Nice

Comment (wlmyng) [reply crates/sui-cluster-test/src/cluster.rs:209]: Suggestion: Maybe a todo here for when sui-test-validator is able to support data persistence?

**Review by wlmyng — APPROVED** [review]
:nice:

File: crates/sui-cluster-test/src/cluster.rs (hunk 1 above)
Comment (gegaowp) [reply crates/sui-cluster-test/src/cluster.rs:209]: I think it should not, iiuc test cluster is an ephemeral service for testing etc. so it should always start fresh, but tbh the change here is just to make compiler happy, so I will leave the decision to @666lcz

**Pull request merged**